Library/Styles/SettingsStyles.swift

```diff
@@ -79,3 +79,85 @@ public func settingsAttributedPlaceholder(_ string: String) -> NSAttributedStrin
     attributes: [NSAttributedString.Key.foregroundColor: UIColor.ksr_text_dark_grey_400]
   )
 }
+
+public func settingsContentViewStyle(_ view: UIView) -> UIView {
```

Comment:
Since we've decided to move away from using lenses I've used key paths and functions.

Also ordered alphabetically.

Reply:
(didn't touch the previous styles for now)